source/plugin/entities/modifying.rst

```diff
@@ -0,0 +1,80 @@
+===================
+Modifying an Entity
+===================
+
+Sure, spawning a regular-old entity is nice and all, but there has to be something more interesting than that? This is
+where ``DataManipulator``\s come into play. An ``Entity`` is a ``DataHolder``, which means that our ``Entity`` can hold
+data. More on ``DataHolder``\s can be found in the :doc:`data documentation <../data/index>`.
+
+``DataManipulator``\s that apply to ``Entity``\s are things such as ``FoodData`` or ``HealthData``.
+A list of applicable ``DataManipulator``\s can be found at ``org.spongepowered.api.data.manipulator.mutable`` and
+``org.spongepowered.api.data.manipulator.mutable.entity``.
+
+Entity Type
+~~~~~~~~~~~
+
+Before we can jump behind the wheel with our ``Entity``, we should check what type of ``Entity`` it is, as we may
+receive an ``Entity`` we didn't create and thus, do not know it's type. Doing this is a simple ``if()`` check. Here is
+an example of checking if our ``Entity`` is a creeper:
+
+.. code-block:: java
+
+    import org.spongepowered.api.entity.Entity;
+    import org.spongepowered.api.entity.EntityTypes;
+    
+    public boolean isCreeper(Entity entity) {
+        if (entity.equals(EntityTypes.CREEPER)) {
```

Comment:
`entity.getType().equals(EntityTypes.CREEPER)` i think. Also better describe this as an equality check. `if()` check looks wrong for various reasons